mysensors/handler.py

```diff
@@ -286,3 +291,13 @@ def handle_pre_sleep_notification(msg):
         return None
     handle_smartsleep(msg)
     return None
+
+
+@HANDLERS_22.register("I_POST_SLEEP_NOTIFICATION")
```

Comment:
The messages will be processed when receiving the `I_PRE_SLEEP_NOTIFICATION` message. This is according to the MySensors spec and code:
https://github.com/mysensors/MySensors/blob/d88506fbc62dac72d6b8b08c1fa3e179143c4542/core/MySensorsCore.cpp#L691-L694

Reply:
I_PRE_SLEEP_NOTIFICATION works correctly and all good. the problem is i have an actor that goes to sleep for in an hourly interval at daylight and at night, it wakes up and need to receive commands asap (its a battery driven light). So with the I_PRE_SLEEP_NOTIFICATION it queues correctly when it is asleep. but when it wakes up (with I_POST_SLEEP_NOTIFICATION) the buffer is still full and all commands still queueing up - the light will never react to commands until the next I_PRE_SLEEP_NOTIFICATION. I am running my addition at the moment in homeassistant for a few weeks and it works flawlessly. wanted to share my (at the moment a bit dirty, sorry) fix with you guys

Reply:
Ok.